librz/core/cdebug.c

```diff
@@ -590,3 +590,58 @@ RZ_API void rz_debug_map_list_visual(RzDebug *dbg, ut64 addr, const char *input,
 		print_debug_maps_ascii_art(dbg, maps, addr, colors);
 	}
 }
+
+/**
+ * Print all traces
+ * @param dbg core->dbg
+ * @param mode output mode, default RZ_OUTPUT_MODE_STANDARD
+ * @param offset offset of address
+ */
+RZ_API void rz_debug_trace_print(RzDebug *dbg, RzOutputMode mode, ut64 offset) {
```

Comment:
please directly pass `RzCmdStateOutput` here instead of just the `mode`, because if in the future we also want to implement the JSON or TABLE output, we are going to need the `PJ` or `RzTable` objects that are inside the `state`.